scripts/lint_rfcs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
we should add a CI job to run our shell scripts through shell check (maybe worth doing here)